### PR TITLE
Fix Keylime workload packages

### PR DESCRIPTION
**Pull request by ansasaki**
Remove all dependencies that are not unique to Keylime

### CODE REVIEW

**Review by sergio-correia — APPROVED** [review]

**Comment by yselkowitz** [comment]
For those dependencies which are maintained by other teams (e.g. openssl) I can understand this, but for those dependencies that are unique to keylime and therefore are (presumably) maintained by the same team, I don't see how this is an improvement.

**Comment by ansasaki** [comment]
> For those dependencies which are maintained by other teams (e.g. openssl) I can understand this, but for those dependencies that are unique to keylime and therefore are (presumably) maintained by the same team, I don't see how this is an improvement.

The improvement I see is a clear message on which packages are essential to us.

If in future some dependencies are dropped or replaced upstream, we can easily follow without the risk of ending up having to maintain a package that we do not use. The fact that there are packages that are brought as dependency by Keylime, and that are only used by Keylime, is circumstantial, we do not care deeply about them. Of course we will keep maintaining such packages, but it is better to clarify which are the packages that really matter to us.

**Comment by yselkowitz** [comment]
@bstinsonmhk I have mixed feelings about this; thoughts?

**Comment by yselkowitz** [comment]
ping @bstinsonmhk

**Comment by yselkowitz** [comment]
Dependencies that are unique to keylime and IMO should remain listed for proper accounting:
```
    - python3-alembic
    - python3-lark
    - python3-sqlalchemy
    - python3-tornado
```

File: configs/sst_security_special_projects-keylime.yaml [hunk 1]
```diff
@@ -8,23 +8,6 @@ data:
   packages:
     - keylime
     - keylime-agent-rust
```
Comment (yselkowitz) [inline configs/sst_security_special_projects-keylime.yaml:10]: @bstinsonmhk has agreed that the dependencies which are unique to keylime, and therefore by definition are being maintained by the same SST, should continue to be tracked here.
```suggestion
    - keylime-agent-rust
    - python3-alembic
    - python3-lark
    - python3-sqlalchemy
    - python3-tornado
```

**Review by yselkowitz — CHANGES_REQUESTED** [review]

File: configs/sst_security_special_projects-keylime.yaml (hunk 1 above)
Comment (ansasaki) [reply configs/sst_security_special_projects-keylime.yaml:10]: OK, done

**Review by yselkowitz — APPROVED** [review]

**Pull request merged**